app/_src/mesh/features/provenance-verification-binaries.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Verify Build Provenance for Kong Mesh Binaries
+badge: enterprise
+---
+
+Starting with 2.8.0, {{site.mesh_product_name}} produces build provenance for binary artifacts, which can be verified using `cosign` / `slsa-verifier` with attestations published to a Docker Hub repository.
```

Comment:
There's no cosign guide after

Reply:
removed the `cosign` method. Only done via `slsa-verifier`